### PR TITLE
Update dashboard-metrics-relation.md

### PR DESCRIPTION
**Pull request by lilyjazz**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dashboard/dashboard-cluster-info.md [hunk 1]
```diff
@@ -49,22 +49,39 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-cluster-info/']
...
-- CPU 使用率：主机当前 1 秒的用户态和内核态 CPU 使用率
-- 物理内存：主机总计的物理内存大小
-- 内存使用率：主机当前内存使用率
-- 部署磁盘：主机上运行实例所在磁盘的文件系统和磁盘挂载路径
-- 磁盘使用率：主机上运行实例所在磁盘的空间使用率
+- CPU 使用率 (CPU Usage)：主机当前 1 秒的用户态和内核态 CPU 使用率
+- 物理内存 (Memory)：主机总计的物理内存大小
+- 内存使用率 (Memory Usage)：主机当前内存使用率
+- 部署磁盘 (Mount Directory)：主机上运行实例所在磁盘的文件系统和磁盘挂载路径
+- 磁盘使用率 (Disk Usage)：主机上运行实例所在磁盘的空间使用率
```
Comment (Oreoxmt) [inline dashboard/dashboard-cluster-info.md:66]:
```suggestion
```

**Review by Oreoxmt — APPROVED** [review]
LGTM

File: dashboard/dashboard-key-visualizer.md [hunk 2]
```diff
@@ -86,12 +86,12 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-key-visualizer/','/docs-cn/dev/how-t
...
 
 * 在热力图中向上或向下滚动。
-* 点击 **Select & Zoom**（框选）按钮，然后点击并拖动以选择要放大的区域。
+* 点击 **框选** (Select & Zoom) 按钮，然后点击并拖动以选择要放大的区域。
 
 ![框选](/media/dashboard/dashboard-keyviz-select-zoom.gif)
 
-* 点击 **Reset**（重置）按钮，将 Region 范围重置为整个数据库。
-* 点击「时间选择框」（界面的 **6 hours** 处），重新选择观察时间段。
+* 点击 **重置** (Reset) 按钮，将 Region 范围重置为整个数据库。
```
Comment (shichun-0415) [inline dashboard/dashboard-key-visualizer.md:93]:
```suggestion
* 点击**重置** (Reset) 按钮，将 Region 范围重置为整个数据库。
```

File: dashboard/dashboard-key-visualizer.md [hunk 3]
```diff
@@ -101,7 +101,7 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-key-visualizer/','/docs-cn/dev/how-t
 
 ### 调整亮度
 
-热力图使用颜色的明暗来表达一个 Bucket 的流量高低，颜色越暗 (cold) 表示该区域的 Region 在这个时间段上读写流量较低，颜色越亮 (hot) 表示读写流量越高，即越热。如果热力图中的颜色太亮或太暗，则可能很难观察到细节。此时，可以点击 **Brightness**（调整亮度）按钮，然后通过滑块来调节页面的亮度。
+热力图使用颜色的明暗来表达一个 Bucket 的流量高低，颜色越暗 (cold) 表示该区域的 Region 在这个时间段上读写流量较低，颜色越亮 (hot) 表示读写流量越高，即越热。如果热力图中的颜色太亮或太暗，则可能很难观察到细节。此时，可以点击 **调整亮度** (Brightness) 按钮，然后通过滑块来调节页面的亮度。
```
Comment (shichun-0415) [inline dashboard/dashboard-key-visualizer.md:104]:
```suggestion
热力图使用颜色的明暗来表达一个 Bucket 的流量高低，颜色越暗 (cold) 表示该区域的 Region 在这个时间段上读写流量较低，颜色越亮 (hot) 表示读写流量越高，即越热。如果热力图中的颜色太亮或太暗，则可能很难观察到细节。此时，可以点击**调整亮度** (Brightness) 按钮，然后通过滑块来调节页面的亮度。
```

File: dashboard/dashboard-key-visualizer.md [hunk 4]
```diff
@@ -121,7 +121,7 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-key-visualizer/','/docs-cn/dev/how-t
 
 ### 刷新与自动刷新
 
-可以通过点击 **Refresh**（刷新）按钮来重新获得基于当前时间的热力图。当需要实时观察数据库的流量分布情况时，可以点击按钮右侧的向下箭头，选择一个固定的时间间隔让热力图按此间隔自动刷新。
+可以通过点击 **刷新** (Refresh) 按钮来重新获得基于当前时间的热力图。当需要实时观察数据库的流量分布情况时，可以点击按钮右侧的向下箭头，选择一个固定的时间间隔让热力图按此间隔自动刷新。
```
Comment (shichun-0415) [inline dashboard/dashboard-key-visualizer.md:124]:
```suggestion
可以通过点击**刷新** (Refresh) 按钮来重新获得基于当前时间的热力图。当需要实时观察数据库的流量分布情况时，可以点击按钮右侧的向下箭头，选择一个固定的时间间隔让热力图按此间隔自动刷新。
```

**Review by shichun-0415 — APPROVED** [review]
Rest LGTM

File: dashboard/dashboard-cluster-info.md [hunk 5]
```diff
@@ -49,22 +49,37 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-cluster-info/']
 
 ## 主机列表
 
-点击**主机**可查看主机列表：
+点击**主机** (Hosts) 可查看主机列表：
```
[the remark this answers is not in the document]
Comment (Oreoxmt) [reply dashboard/dashboard-cluster-info.md:52]: Done https://github.com/pingcap/docs-cn/pull/12385/files#diff-f4f3271c05742c5b454763e67ed39bb4caf64a68fcf50c20b1638d56c5996c8e

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**